src/main/functions/DIDResolution.ts

```diff
@@ -31,6 +31,42 @@ export function getResolver(opts: ResolveOpts): Resolvable {
   }
 }
 
+export function getResolverUnion(subjectSyntaxTypesSupported: string[] | string, resolverMap: Map<string, Resolvable>): Resolvable {
```

Comment:
I rather would have seen this as part of a class/object that also contains the map on the RP/OP builder side

Reply:
changed the impl of `getResolverUnion`. now we have two separate method: `mergeAllDidMethods` which returns the content that should end up in `subject_syntax_types_supported`. But having this in OP/RP will force us to duplicate codes for this